docs/components/Select2View.jsx

```diff
@@ -150,21 +150,65 @@ export default class Select2View extends React.PureComponent {
       <PropDocumentation
         title="<Select2 /> Props"
         availableProps={[
-          {
-            name: "children",
-            type: "React.Node",
-            description: "Select2 content.",
-          },
           {
             name: "className",
             type: "string",
             description: "Optional additional CSS class name to apply to the container.",
             optional: true,
           },
           {
-            name: "onPerformAction",
-            type: "Function",
-            description: "Handler function for the performAction event.",
+            name: "name",
+            type: "String",
+            description:
+              "Name for input element which will also be used as the id for the label to reference",
+          },
+          {
+            name: "label",
+            type: "React Node",
+            description: "Label text",
+          },
+          {
+            name: "hideLabel",
+            type: "Boolean",
+            description:
+              "Hide label for visual purposes only (will still be available to screen readers)",
+            optional: true,
+          },
+          {
+            name: "options",
+            type: <code>{"Array<{ value: string, content?: ReactNode }>"}</code>,
+            description:
+              "List of options to be selected. 'value' is a react node for custom rendering, but you must have a stringValue for searchability",
```

Comment:
This description looks like it's referencing the old `option`

Reply:
zoinks thx